app/src/main/java/com/github/sdp/ratemyepfl/viewmodel/RoomReviewsListViewModel.kt

```diff
@@ -37,4 +37,8 @@ class RoomReviewsListViewModel @Inject constructor(
     fun getReviews(): LiveData<List<Review?>> {
         return reviewsLiveData
     }
+
+    fun refreshReviewsList() {
```

Comment:
Why make updateReviewList private and then use this function to call it ?

Reply:
I agree, this adds some boilerplate ^^